lib/Doctrine/ruleset.xml

```diff
@@ -37,7 +37,33 @@
     <!-- Forbid deprecated functions -->
     <rule ref="Generic.PHP.DeprecatedFunctions"/>
     <!-- Forbid alias functions, i.e. `sizeof()`, `delete()` -->
-    <rule ref="Generic.PHP.ForbiddenFunctions"/>
+    <rule ref="Generic.PHP.ForbiddenFunctions">
+        <properties>
+            <property
+                name="forbiddenFunctions"
+                type="array"
+                value="
+                    chop => rtrim,
+                    close => closedir,
+                    delete => unset,
+                    doubleval => floatval,
+                    fputs => fwrite,
+                    ini_alter => ini_set,
+                    is_double => is_float,
+                    is_integer => is_int,
+                    is_long => is_int,
+                    is_null => null,
```

Comment:
Is this right?

Reply:
I suppose it just forbids using `is_null` without suggesting an alternative function since you should be using `!== null`.

Reply:
Yes, if `null` is provided, then the function is forbidden without alias being suggested, i.e.:
```
 11 | ERROR | The use of function chop() is forbidden; use rtrim() instead
 12 | ERROR | The use of function sizeof() is forbidden; use count() instead
 13 | ERROR | The use of function is_null() is forbidden
```